src/Symfony/Component/Worker/Consumer/ConsumerInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Worker\Consumer;
+
+use Symfony\Component\Worker\MessageCollection;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+interface ConsumerInterface
```

Comment:
I see that you have not provided an implementation of `ConsumerInterface`. How come?